src/common/Loading/index.js

```diff
@@ -0,0 +1,9 @@
+import { Wrapper, StyledSpiner } from "./styled";
+
+const Loading = () => {
+    <Wrapper>
+        <StyledSpiner />
+    </Wrapper>
```

Comment:
Czy nie lepiej byłoby zrobić podobnie jak zrobiłeś w "NoResults", żeby dodać od razu ten tekst "Search results for [...]"? Potem w zależności od stanu (loading/success/no result/error) wywołalibyśmy od razu cały komponent z tekstem i grafiką/animacją.

Reply:
Mój błąd, źle tu napisałem, bo ten tekst "Search results for [...]" musi być cały czas na ekranie podczas wyszukiwania, więc on raczej będzie osobno, a to co jest pod tym tekstem będzie się zmieniało.

Reply:
> Czy nie lepiej byłoby zrobić podobnie jak zrobiłeś w "NoResults", żeby dodać od razu ten tekst "Search results for [...]"? Potem w zależności od stanu (loading/success/no result/error) wywołalibyśmy od razu cały komponent z tekstem i grafiką/animacją.

Mój błąd, tu chyba źle napisałem, bo ten tekst "Search results for [...]" będzie cały czas na ekranie podczas wyszukiwania, więc on raczej będzie osobno, a to co jest pod tym tekstem będzie się zmieniało.